spec/services/types_generator_spec.rb

```diff
@@ -250,7 +272,12 @@
           .sort
           .uniq
       end
-      let(:known_genreless) { ['Policy brief', 'Speaker notes'] }
+      # these represent subtypes that will get the genre from the parent type
+      let(:known_genreless) do
+        ['Policy brief', 'Speaker notes', '3D model', 'Book chapter', 'Broadcast',
+         'Conference session', 'Other spoken word', 'Presentation recording',
+         'Presentation slides', 'Text']
+      end
```

Comment:
We do not have to have a genre mapping for all subtypes, as it will get the genre from the top level mapping.  We only need a subtype genre configured if it is additive to the main type.